app/templates/components/forms/session-speaker-form.hbs

```diff
@@ -53,6 +53,7 @@
               {{/ui-dropdown}}
             {{else}}
               {{#ui-dropdown class='search selection' onChange=(action (mut (get data.session field.fieldIdentifier))) as |execute mapper|}}
+                {{input type='hidden' name=(if field.isRequired (concat 'session_' field.fieldIdentifier '_required') (concat 'session_' field.fieldIdentifier))}}
```

Comment:
shouldn't the input have a value?

Reply:
@CosmicCoder96 This input is just created so that the `Dropdown` can be assigned a name which can be used for validators , Assigning a value to it will raise unwanted errors

Reply:
@kushthedude And what will it validate if it has nothing in the input?

Reply:
Also, what unwanted errors?